Formula/ncmpc.rb

```diff
@@ -27,20 +27,12 @@ class Ncmpc < Formula
   depends_on "libmpdclient"
   depends_on "pcre2"
 
-  on_linux do
-    depends_on "gcc"
-  end
-
   fails_with gcc: "5"
 
   def install
-    mkdir "build" do
-      system "meson", *std_meson_args, "-Dcolors=false",
-                                       "-Dnls=disabled",
-                                       "-Dregex=enabled",
-                                       ".."
-      system "ninja", "install"
-    end
+    system "meson", "setup", "build", "-Dcolors=false", "-Dnls=enabled", "-Dregex=enabled", *std_meson_args
```

Comment:
Enabled `nls`. We already have a `gettext` dependency. If that suffices, enabling it doesn't seem to hurt.

Reply:
Should we revision bump if we do that?

Reply:
I'd rather not. No one seems to have missed it, so I wouldn't consider existing bottle installs broken enough to be worth forcing a reinstall for everyone.